### ISSUE TITLE
Some questions about the train_2ddense.py

### ISSUE THREAD

**Issue by WdBlink** [xmengli999/H-DenseUNet#16]
Hello, please ask line 62 in the train_2ddense.py code:
```
crop_img = img[a - deps / 2:a + deps / 2, b - rows / 2:b + rows / 2,
                c - cols / 2: c + cols / 2 + 1].copy()
cropp_tumor = tumor[a - deps / 2:a + deps / 2, b - rows / 2:b + rows / 2,
                  c - cols / 2:c + cols / 2 + 1].copy()
```
What function does this code play? The colon here in python: What kind of arithmetic operation is it?

**Comment by Apple-zly** [comment]
Hello,  I run "train_2ddenseunet.py",  The program is killed as soon as it runs. I don't know this is why?

**Comment by xmengli999** [comment]
check if the CPU memory is full when running the program.(CPU memory on my server is 46G. ) At the beginning, the program load all data at once.

**Comment by Excelsiorwu** [comment]
Hello,I can  not find   'long skip'  in the Up-sampling process ,which your  paper has mentioned. i wonder that  the Up-sampling process only take the  output of dense encoder without long skip from the front layers.

**Comment by xmengli999** [comment]
Yes, there are two version, One is densenet and the other is the densenet with u net connections. Two versions are available. Both of  it can be used in the hybrid architecture.
